janitor/utils.py

```diff
@@ -31,11 +34,14 @@ def check(varname: str, value, expected_types: list):
     """
     is_expected_type = False
     for t in expected_types:
-        if isinstance(value, t):
-            is_expected_type = True
+        if t is callable:
+            is_expected_type = t(value)
+        else:
+            is_expected_type = isinstance(value, t)
+        if is_expected_type is True:
```

Comment:
I think this can be simplified:

```python
if is_expected_type:
    break
```

This can be coupled with an in-line type annotation first declaring `is_expected_type`:

```python
is_expected_type: bool = False
```